Source/Core/Updater/UI.cpp

```diff
@@ -2,13 +2,14 @@
 // Licensed under GPLv2+
 // Refer to the license.txt file included.
 
-#include "Updater/UI.h"
+#include "UpdaterCommon/UI.h"
```

Comment:
Since there is now a common updater UI, should this file be called something like Win32UI, or WindowsUI, or something like that?